runtime/doc/lsp.txt

```diff
@@ -755,6 +755,12 @@ start_client({config})                                *vim.lsp.start_client()*
                                          array.
                     {handlers}         Map of language server method names to
                                        |lsp-handler|
+                    {settings}         Map with case-sensitive keys to
+                                       'workspace/configuration' event
+                                       responses. These are sent to the server
+                                       via `workspace/didChangeConfiguration`.
+                                       If settings are changed, the client must 
+                                       be notified with `client.workspace_did_change_configuration({settings})`.
```

Comment:
```suggestion
                    {settings}         Map with language server specific
                                       settings. These are returned to the language server if
                                       requested via `workspace/configuration`. Keys are
                                       case-sensitive.
```


I'd trim this down to the `workspace/configuration` part for now given that the `didChangeConfiguration` part is still in nvim-lspconfig and not in core.